tools/cli/completion.bash

```
@@ -0,0 +1,93 @@
+  local meteor_cmd="${COMP_WORDS[0]}"
+  local completion_status
+
+  _meteor_completion_log "bash start cmd=${meteor_cmd} words=${COMP_WORDS[*]} index=${COMP_CWORD}"
+
+  cur="${COMP_WORDS[COMP_CWORD]}"
+  prev="${COMP_WORDS[COMP_CWORD-1]}"
+
+  if [ "$COMP_CWORD" -eq 1 ] && [[ "$cur" != -* ]]; then
+    local oldifs=$IFS
+    IFS=$'\n'
+    COMPREPLY=( $(compgen -W "$(printf '%s\n' "${static_top_level_commands[@]}")" -- "$cur") )
```

Shellcheck (0.11.0)

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)